server/controllers/wazuh-reporting.ts

```diff
@@ -154,7 +146,7 @@ export class WazuhReportingCtrl {
           printer,
           isAgents,
           apiId,
-          section === 'groupConfig' ? tab : false
+          section === 'groupConfig' ? tab : null
```

Comment:
Be careful using `null`

`false` would be more appropriate, unless there is justified reason to use `null` instead.

Reply:
That parameter can be a group ID ( `string` ) so using a boolean could be confusing, but if its value is managed correctly we could use any falsy values. I considered changing it to `null`.